.github/BOTMETA.yml

```diff
@@ -901,6 +902,8 @@ files:
   $module_utils/junos.py:
     maintainers: $team_networking
     labels: networking
+  $module_utils/manageiq.py:
+    maintainers: $team_manageiq
```

Comment:
wasn't sure if a label was required here... we don't have one for remote_management anyways.